ocaml/xapi/livepatch.ml

```diff
@@ -88,7 +88,7 @@ module BuildId = struct
 end
 
 module KernelLivePatch = struct
-  let get_running_livepatch () =
+  let get_running_livepatch' s =
```

Comment:
The original function (now below) called a script and then processed the output, so did more "getting". The "running" bit seems important though, because it looks for already loaded/applied/running livepatches.

Reply:
This function does nothing about running - it is extracting a version from a string. The string might come from a running system, though. So it would be at the call site where "running" should appear.

Reply:
It does if you look at the tests: it only takes things from the string (which is the output from a command) that indicates applied patches and ignores the not-yet-applied ones.

Reply:
Fair; my argument was, that this is a string function and not a function that obtains something from the running system. But you can argue that "running" is still a property of the version that this function is extracting. We can keep this.